### PR TITLE
skip adding show and describe

### PR DESCRIPTION
**Pull request by niladrem**

### CODE REVIEW

File: streamingcli/jupyter/notebook_converter.py [hunk 1]
```diff
@@ -201,6 +201,10 @@ def _render_flink_app(notebook_entries: Sequence[NotebookEntry]) -> ConvertedNot
         return ConvertedNotebook(content=autopep8.fix_code(flink_app_script), remote_jars=list(remote_jars),
                                  local_jars=list(local_jars))
 
+    @staticmethod
+    def _skip_statement(statement: str) -> bool:
+        return statement.lower().startswith(('select', 'show', 'desc'))
```
Comment (swtwsk) [inline streamingcli/jupyter/notebook_converter.py:206]: I would also add `DESCRIBE` and `EXPLAIN`

**Review by swtwsk — CHANGES_REQUESTED** [review]

File: streamingcli/jupyter/notebook_converter.py (hunk 1 above)
Comment (maver1ck) [reply streamingcli/jupyter/notebook_converter.py:206]: Can we use is_dql from magics ?

Comment (niladrem) [reply streamingcli/jupyter/notebook_converter.py]: I think is_dql is a bit too complex for this case. We would have to declare extra dicts and a method to have the same logic in more than 1 line.

Comment (maver1ck) [reply streamingcli/jupyter/notebook_converter.py]: Hmmm. With your solution this statement will be included (whitespace at the beginning)
```
    SELECT xxx...
```

**Review by maver1ck — APPROVED** [review]

**Review by swtwsk — APPROVED** [review]

**Pull request merged**